### ISSUE TITLE
[3.6.0-beta.2] Closing a panel tab and re-opening it returns json

### ISSUE THREAD

**Issue by medienbaecker** [k-next/kirby#3734]
**Describe the bug**  
I re-opened a panel tab by pressing ⌘⇧T and got a weird json response instead of the panel.

**To Reproduce**  
1. Open a panel
2. Close the tab
3. Re-open the tab

**Screenshots**  

<img width="1424" alt="Screenshot-30-09-2021_14-24-21@2x" src="https://user-images.githubusercontent.com/7975568/135454313-0bce5087-ff5c-4622-aeb9-a7164d83255f.png">

**Kirby Version**  
3.6.0 beta 2

**Desktop (please complete the following information):**  
 - OS: macOS 11.5.2
 - Browser: Chrome
 - Version: 94.0.4606.61

**Comment by bastianallgeier** [comment]
Interesting 🤔

**Comment by distantnative** [comment]
@bastianallgeier i think that's the same we had with the back button and Fiber

**Comment by afbora** [comment]
I guess similar issue with https://github.com/getkirby/kirby/issues/3469

**Comment by bastianallgeier** [comment]
I'm still totally lost why this happens. It's also super hard to debug because you cannot use the console for this.

**Comment by afbora** [comment]
I couldn't find anything concrete either. But I noticed this: After refreshing the page, this issue does not happen. It occurs when the tab is closed and reopened `after the fiber request`.

**Comment by bastianallgeier** [comment]
I checked and it's definitely a Chrome issue. Firefox and Safari handle this correctly. Chrome somehow seems to be storing the last fetch event and recalling that. Since our last fetch is a Fiber fetch, the json response is being rendered. I really don't know if we could and should fix this at all.

**Comment by afbora** [comment]
In similar situation I get `Unauthenticated` error in Firefox. Could it have something to do with it?

https://user-images.githubusercontent.com/3393422/136042782-40f5d89e-0724-4aed-b3a5-ceb02759c18b.mp4

Also getting `Invalid CSRF token` error after login redirection.

**Comment by afbora** [comment]
I've researched a bit and I think the issue is about response caching. I found a thread on SO. I tried the solution and I can say it works.

https://stackoverflow.com/questions/9445588/re-open-last-closed-tab-causing-to-show-last-ajax-request-content

To give you idea, I've put following codes to top of `/kirby/src/Panel/Panel.php` file and works:

````php
header("Cache-Control: no-store, no-cache, must-revalidate, max-age=0");
header("Cache-Control: post-check=0, pre-check=0", false);
header("Pragma: no-cache");
````
